database/repositories/helper.py

```diff
@@ -40,3 +39,15 @@ def get_whole_rank(self) -> list:
         users = session.execute(select(self.user_model).order_by(self.user_model.points.desc())).scalars().all()
         session.close()
         return users
+
+    def get_reward(self, emoji_name) -> int:
+        session = self.sessionmaker()
+        reward = session.execute(select(self.helper_reward_model.reward).where(self.helper_reward_model.emoji_name == emoji_name)).scalars().first()
+        session.close()
+        return reward
+
+    def get_emojis(self) -> list:
+        session = self.sessionmaker()
+        emojis = session.execute(select(self.helper_reward_model.emoji_name)).scalars().all()
+        session.close()
+        return emojis
```

Comment:
daj tego newline'a od razu, bo prędzej czy później i tak jakiś reformat automatyczny go doda, a tym sposobem nie zakradnie się to do innego PRa z dupy